x-pack/plugins/aiops/public/components/explain_log_rate_spikes/explain_log_rate_spikes_wrapper.tsx

```diff
@@ -26,33 +24,104 @@ import {
 import type { WindowParameters } from '@kbn/aiops-utils';
 import type { DataView } from '@kbn/data-views-plugin/public';
 
-import {
-  Accessor,
-  Dictionary,
-  parseUrlState,
-  Provider as UrlStateContextProvider,
-  isRisonSerializationRequired,
-  getNestedProperty,
-  SetUrlState,
-} from '../../hooks/url_state';
 import { useData } from '../../hooks/use_data';
-import { useUrlState } from '../../hooks/url_state';
+import { useUrlState, usePageUrlState, AppStateKey } from '../../hooks/url_state';
 
 import { FullTimeRangeSelector } from '../full_time_range_selector';
 import { DocumentCountContent } from '../document_count_content/document_count_content';
 import { DatePickerWrapper } from '../date_picker_wrapper';
 
 import { ExplainLogRateSpikes } from './explain_log_rate_spikes';
+import { SearchPanel } from '../search_panel';
+import { SEARCH_QUERY_LANGUAGE, SearchQueryLanguage } from '../../../common/types';
+import { useAiOpsKibana } from '../../kibana_context';
 
 export interface ExplainLogRateSpikesWrapperProps {
   /** The data view to analyze. */
   dataView: DataView;
 }
 
+const defaultSearchQuery = {
+  match_all: {},
+};
+
+export interface AiOpsIndexBasedAppState {
+  searchString?: Query['query'];
+  searchQuery?: Query['query'];
+  searchQueryLanguage: SearchQueryLanguage;
+  filters?: Filter[];
+}
+
+export const getDefaultAiOpsListState = (
+  overrides?: Partial<AiOpsIndexBasedAppState>
+): Required<AiOpsIndexBasedAppState> => ({
+  searchString: '',
+  searchQuery: defaultSearchQuery,
+  searchQueryLanguage: SEARCH_QUERY_LANGUAGE.KUERY,
+  filters: [],
+  ...overrides,
+});
+
+const restorableDefaults = getDefaultAiOpsListState();
+
 export const ExplainLogRateSpikesWrapper: FC<ExplainLogRateSpikesWrapperProps> = ({ dataView }) => {
+  const { services } = useAiOpsKibana();
+  const { notifications, data } = services;
+  const { toasts } = notifications;
+
+  const [aiopsListState, setAiopsListState] = usePageUrlState(AppStateKey, restorableDefaults);
   const [globalState, setGlobalState] = useUrlState('_g');
 
-  const { docStats, timefilter } = useData(dataView, setGlobalState);
+  useEffect(() => {
+    if (!dataView.isTimeBased()) {
```

Comment:
Is this condition ever triggered? I still see a blank page if selecting a data view without a time field. The warning message will need changing as currently it refers to anomaly detection.